module3/lessons/many-to-many.md

```diff
@@ -0,0 +1,608 @@
+---
+title: Many to Many Relationships
+tags: database, schema, relationships
+---
+
+## Learning Goals
+
+- Describe the relationship between a foreign key on one table and a primary key on another table
+- Diagram a one-to-many relationship
+- Diagram a many-to-many relationship
+- Understand what a join table is and why we need one
+- Create many-to-many relationships in Rails
+- Interpret `schema.rb`
+
+## Vocabulary
+
+- Foreign Key
+- Primary Key
+- One-to-Many Relationship
+- Many-to-Many Relationship
+- Join Table
+- Migration
+- Schema
+
+## Setup
+
+TODO: update starter repo branch for song tests with song serializer
+
+You can begin this lesson on the `many-to-many-setup` branch of the Set List Tutorial Repo [here](https://github.com/turingschool-examples/set-list-api/tree/many-to-many-setup).
```

Comment:
I think this should be many-to-many-starter right?

Reply:
Yes, thank you!